README.md

```diff
@@ -13,7 +13,9 @@ This repo contains samples that demonstrate configuring PreEmptive Protection JS
 - [Electron](Electron): demonstrates how you can use the JSDefender Webpack plugin to protect Electron projects.
 - [Angular](Angular): demonstrates how you can use the JSDefender Webpack plugin to protect Angular projects.
 - [Ionic](Ionic): demonstrates how you can use the JSDefender Webpack plugin to protect different Ionic projects.
+- [CLI/Web](CLI/Web): demonstrates how you can use the JSDefender Webpack plugin to protect Vanilla JavaScript projects.
```

Comment:
@NickyArias use this instead (feel free to fix grammatical or wording issues):
[CLI](CLI): demonstrates how you can use the JSDefender CLI to protect JavaScript projects with multiple input files e.g. a Vanilla JavaScript Web app without any bundler.

Reply:
@NickyArias you forgot to update the beginning: `[CLI](CLI)`